ui/src/serverIO.js

```diff
@@ -329,22 +327,12 @@ class ServerIO {
         }
 
         case 'loadReady': {
-          this.dispatch(
-            setLoading(false, 'SC Ready', record.message, true, () =>
-              this.dispatch(stopQueue()),
-            ),
-          );
-
+          this.dispatch(hideWaitDialog());
           break;
         }
 
         case 'inSafeArea': {
-          this.dispatch(
-            setLoading(false, 'SC Safe', record.message, true, () =>
-              this.dispatch(stopQueue()),
-            ),
-          );
-
+          this.dispatch(hideWaitDialog());
```

Comment:
Same same, but this line was introduced [here](https://github.com/mxcube/mxcubeweb/pull/807/files#diff-3ce702edf4adc9b09256a43ff92b1001d01e8ca66e6d547a52a587f4a5ffb580R218).